apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/BigtableFixture.cs

```diff
@@ -200,9 +203,16 @@ public BigtableFixture()
 
         public void Dispose()
         {
-            if (TableAdminClient != null && TableName != null)
+            if (TableAdminClient != null)
             {
-                TableAdminClient.DeleteTable(new DeleteTableRequest { Name = TableName.ToString() });
+                foreach (var tableName in CreatedTables)
+                {
+                    try
+                    {
+                        TableAdminClient.DeleteTable(new DeleteTableRequest { Name = tableName.ToString() });
```

Comment:
Can you not do `DeleteTable(TableName)`? If not, that suggests we should probably be fixing that in the GAPIC config.

Reply:
Ah, okay. (We've seen that before, and it's slightly annoying, but there's not much we can do about it as we don't want dependencies in either direction...)